nextflow.config

```diff
@@ -7,86 +7,92 @@
-  qiime_timezone = 'Europe/Berlin'
+  input             = false
+  pacbio            = false
+  FW_primer         = false
+  RV_primer         = false 
+  classifier        = false
+  metadata          = false
 
   // Defines all parameters that are independent of a test run
-  trunc_qmin = 25 //to calculate params.trunclenf and params.trunclenr automatically
-  trunc_rmin = 0.75 //to calculate params.trunclenf and params.trunclenr automatically
-  trunclenf = false
-  trunclenr = false
-  maxEE = 2
-  maxLen = 2999 //2999 is the maximum allowed read length in dada2 version 1.12
-  minLen = 50
+  extension         = "/*_R{1,2}_001.fastq.gz"
+  help              = false
+  outdir            = './results'
+  publish_dir_mode  = 'copy'
+  tracedir          = "${params.outdir}/pipeline_info"
+  clusterOptions    = false
+  trunc_qmin        = 25
+  trunc_rmin        = 0.75
+  trunclenf         = false
+  trunclenr         = false
+  maxEE             = 2
+  maxLen            = "Inf"
+  minLen            = 50
   metadata_category = false
-  double_primer = false
-  retain_untrimmed = false
-  exclude_taxa = "mitochondria,chloroplast"
-  keepIntermediates = false
-  classifier_removeHash = false
-  min_frequency = false
-  min_samples = false
+  double_primer     = false
+  retain_untrimmed  = false
+  exclude_taxa      = "mitochondria,chloroplast"
+  min_frequency     = 1
+  min_samples       = 1
   multipleSequencingRuns = false
-  phred64 = false
-  split = "-"
-  skip_fastqc = false
+  single_end        = false
+  sample_inference  = "pooled"
+  illumina_pe_its   = false
+  concatenate_reads = false
+
+  //skipping steps
+  skip_fastqc       = false
+  skip_alpha_rarefaction = false
+  skip_abundance_tables = false
+	skip_barplot      = false
+	skip_taxonomy     = false
+	skip_alpha_rarefaction = false
+	skip_diversity_indices = false
+	skip_ancom        = false
+  skip_multiqc      = false
 
   //Database specific parameters
   //currently only this is compatible with process make_SILVA_132_16S_classifier
   reference_database = "https://www.arb-silva.de/fileadmin/silva_databases/qiime/Silva_132_release.zip"
-  dereplication = 99
-  taxon_reference = "silva"
+  dereplication     = 99
+  taxon_reference   = "silva"
+  dada_ref_taxonomy = false
+  dada_ref_species  = false
 
   // Boilerplate options
-  name = false
-  multiqc_config = false
-  email = false
-  email_on_fail = false
+  enable_conda      = false
+  name              = false
+  multiqc_config    = ''
+  multiqc_title     = ''
+  email             = false
+  email_on_fail     = false
   max_multiqc_email_size = 25.MB
-  plaintext_email = false
-  monochrome_logs = false
-  help = false
-  tracedir = "${params.outdir}/pipeline_info"
-  awsqueue = false
-  awsregion = 'eu-west-1'
+  plaintext_email   = false
+  monochrome_logs   = false
+  help              = false
+  tracedir          = "${params.outdir}/pipeline_info"
+  awsqueue          = false
+  awsregion         = 'eu-west-1'
```

Comment:
not sure, others (rnaseq, viralrecon) removed these params because it can be provided via nf-core/configs now

Reply:
Ok, thanks, will remove those as well.